src/definitions.ts

```diff
@@ -151,7 +142,7 @@ export interface AdMobPlugin {
    */
   addListener(
     eventName: 'onAdOpened',
-    listenerFunc: (info: any) => void,
+    listenerFunc: (info: void) => void,
```

Comment:
If `info` does not carry any value, this will be better as:

````
listenerFunc: () => void```
```

Reply:
You are right. I will change this👍